landing-pages/site/content/en/community/_index.html

```diff
@@ -16,10 +16,10 @@
             </p>
             <ul class="ticks-blue mx-auto">
                 <li>
-                    Users list: <a class="list-link" href="mailto:users-subscribe@airflow.apache.org">users-subscribe@airflow.apache.org</a> <small>and <a class="list-link" href="mailto:users-unsubscribe@airflow.apache.org">users-unsubscribe@airflow.apache.org</a> to unsubscribe</small>
+                    Dev list - for discussion, news, and information on Airflow: <a class="list-link" href="mailto:dev-subscribe@airflow.apache.org">dev-subscribe@airflow.apache.org</a> <small>and <a class="list-link" href="mailto:dev-unsubscribe@airflow.apache.org">dev-unsubscribe@airflow.apache.org</a> to unsubscribe</small>
                 </li>
                 <li>
-                    Dev list: <a class="list-link" href="mailto:dev-subscribe@airflow.apache.org">dev-subscribe@airflow.apache.org</a> <small>and <a class="list-link" href="mailto:dev-unsubscribe@airflow.apache.org">dev-unsubscribe@airflow.apache.org</a> to unsubscribe</small>
+                    Users list - for users to ask how-to and troubleshooting questions: <a class="list-link" href="mailto:users-subscribe@airflow.apache.org">users-subscribe@airflow.apache.org</a> <small>and <a class="list-link" href="mailto:users-unsubscribe@airflow.apache.org">users-unsubscribe@airflow.apache.org</a> to unsubscribe</small>
```

Comment:
I'm wondering if it would make sense to add a comment that after subscribing the emails should be send to `dev@airflow.apache.org` 🤔